instance.tf

```diff
@@ -16,13 +16,13 @@ resource "google_sql_database_instance" "instance" {
     user_labels           = local.labels
 
     backup_configuration {
-      enabled                        = local.backup_config.enabled
-      start_time                     = local.backup_config.start_time
-      point_in_time_recovery_enabled = local.backup_config.point_in_time_recovery_enabled
-      location                       = local.backup_config.location
-      transaction_log_retention_days = local.backup_config.transaction_log_retention_days
+      enabled                        = coalesce(var.backup_config.enabled, var.highly_available)
```

Comment:
I feel this may lead to errors if, for some stupid reason, someone would set backup to false and highly available to true. Probably not worth it to capture that here and just have the provider report stupidity... 🤔

Reply:
> coalesce takes any number of arguments and returns the first one that isn't null or an empty string.

How would that be a problem?